docs-source/content/userguide/managing-domains/fmw-infra/_index.md

```diff
@@ -353,19 +353,24 @@ is provided that demonstrates how to create a FMW Infrastructure domain.
 
 There are two kind of patches that can be applied to the FMW Infrastructure binaries:
 
-* ZDT compliant, meaning that the patch can be applied in a rolling fashion.
-* non-ZDT compliant, meaning that the domain must be shutdown and restarted.
-
-If you wish to apply a ZDT compliant patch which can be applied in a rolling
+* Patches which are eligible for Zero Downtime Patching (ZDP), meaning that
+  they can be applied in a rolling fashion.
```

Comment:
applied in a rolling fashion. -> applied with a rolling restart.

Reply:
fixed